src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.cs

```diff
@@ -31,6 +31,7 @@ public partial class Process : Component
         private bool _isRemoteMachine;
         private string _machineName;
         private ProcessInfo? _processInfo;
+        private string? _processName;
```

Comment:
This can also be moved  to Process.Windows.cs.

Reply:
I tried to move it, but got compilation errors.  It's used on line 958 in this file